s2fft/precompute_transforms/construct.py

```diff
@@ -342,10 +350,11 @@ def healpix_phase_shifts(
 
     Returns:
         np.ndarray: Vector of phase shifts with shape :math:`[thetas, 2L-1]`.
+
     """
     thetas = samples.thetas(L, "healpix", nside)
     phase_array = np.zeros((len(thetas), 2 * L - 1), dtype=np.complex128)
-    for t, theta in enumerate(thetas):
+    for t in range(len(thetas)):
```

Comment:
`theta` unused here so I just changed to a loop over `range(len(thetas))`

Reply:
Nice, originally I suspect we were using theta here but then pulled this outside the loop.